.github/workflows/static_analysis_and_tests.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         run: |
           pip install -r requirements.txt
       - name: Set PYTHONPATH environment variable
-        run: echo "PYTHONPATH=${GITHUB_WORKSPACE}/living_documentation_generator/living_documentation_generator" >> $GITHUB_ENV
+        run: echo "PYTHONPATH=${GITHUB_WORKSPACE}/living_documentation_regime/living_documentation_regime" >> $GITHUB_ENV
```

Comment:
This looks weird. Do we still need the variable defined?
Can you test it to remove it?

Reply:
After the removal, it works correctly. Change at: [b09c30f](https://github.com/AbsaOSS/living-doc-generator/pull/72/commits/b09c30fe59702a248d2719b142bcf876e11c03af).